src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/taskmanagementservicetests/CompleteTaskWithPrivilegeAndCompletionOptionsTest.java

```diff
@@ -335,40 +276,6 @@ void should_throw_role_assignment_verification_exception_when_has_access_returns
                 verify(camundaService, times(0)).completeTask(any(), anyBoolean());
             }
 
-            @Test
-            void should_throw_role_assignment_verification_exception_when_has_access_returns_false_gp_flag_off() {
```

Comment:
same as previous comment. we should still be able to test role_assignment_verification exception regardless of gp flag